lib/hsa/hsaBeamformUpchan.cpp

```diff
@@ -1,11 +1,21 @@
 #include "hsaBeamformUpchan.hpp"
 
+#include "Config.hpp"             // for Config
+#include "gpuCommand.hpp"         // for gpuCommandType, gpuCommandType::KERNEL
+#include "hsaDeviceInterface.hpp" // for hsaDeviceInterface
+
+#include <string.h> // for memcpy, memset
+
+namespace kotekan {
+class bufferContainer;
```

Comment:
I realise we aren't actually making use of `bufferContainer` here, but it seems odd to forward declare in a CPP file when we have the header.

Reply:
I also don't understand this. What happened in some cases was:
1. iwyu changes include to fwd declare -> doesn't compile
2. I revert the change and force it to keep the include
3. iwyu insists on having both (might be a bug?)